types/supercluster/index.d.ts

```diff
@@ -33,6 +33,20 @@ export interface Options {
      * Whether timing info should be logged.
      */
     log?: boolean;
+    /**
+     * initial determines the starting values for the accumulator for the reducer.
+     */
+    initial?: () => any;
+    /**
+     * map takes the properties of the geoJson properties of the point and
+     * maps it to a MapProperties that is then passed to the reducer.
+     */
+    map?: (geoJsonProperties: any) => any;
+    /**
+     * reduce takes an accumulator and the mapped value of the properties and
+     * mutates the accumulator based on the value.
+     */
+    reduce?: (acc: any, value: any) => void;
```

Comment:
Typically I try to use the param names as the original source code ([`supercluster/index.js`](https://github.com/mapbox/supercluster/blob/master/index.js#L26-L33)), for this definition the params would be:

- `acc` => `accumulated`
- `geoJsonProperties` => `props`
- `value` => `props`

or pull the param names from the examples in the main README page.

```js
var index = supercluster({
    initial: function() { return {sum: 0}; },
    map: function(props) { return {sum: props.myValue}; },
    reduce: function(accumulated, props) { accumulated.sum += props.sum; }
});
```

Reply:
yea that makes sense will make the change